modules/cudf/src/table/arrow.cpp

```diff
@@ -90,6 +104,7 @@ Napi::Value Table::from_arrow(Napi::CallbackInfo const& info) {
   try {
     auto table = cudf::from_arrow(*arrow_table);
     output.Set("table", Table::New(env, std::move(table)));
+    delete buffer_reader;
```

Comment:
Just realized if `cudf::from_arrow` throws, having this here will cause a memory leak. We should put this in both the `catch {}` clause before `NAPI_THROW`, and again just before `return output`.

Reply:
@bryevdv 

```c++
  try {
    auto table = cudf::from_arrow(*arrow_table);
    output.Set("table", Table::New(env, std::move(table)));
  } catch (std::exception const& e) {
    delete buffer_reader;
    NAPI_THROW(Napi::Error::New(env, e.what()));
  }

  delete buffer_reader;

  return output;
```